### ISSUE TITLE
Sandbox kingdom crash?

### ISSUE THREAD

**Issue by azirok** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#330]
Only error I see is community patch related, but I'm not a scholar at deciphering these crash logs.  Anybody else have this issue?

[rgl_log_errors_2912.txt](https://github.com/Tyler-IN/MnB2-Bannerlord-CommunityPatch/files/4639118/rgl_log_errors_2912.txt)

**Comment by peppermor** [comment]
Its pure vanilla related. Nothing to do with mods

**Comment by azirok** [comment]
yeah, it is not the community patch, it happens regardless for me no matter what/if mods are loaded.  I just thought maybe someone here has run into this.

**Comment by Solganis** [comment]
@azirok they will patch it in hotfix soon.

**Comment by Tyler-IN** [comment]
We'll track it and see if we can figure it out too.

**Comment by exeSleepyViking** [comment]
From my understanding this issue isn't from this mod. If you are using Freeplay Aka Just let me play, then that could be the issue but even in vanilla this is occurring
